src/js/constants.js

```diff
@@ -40,6 +40,17 @@ var exports = {
   TRACKING_THRESHOLD: 3,
   MAX_COOKIE_ENTROPY: 12,
 
+  // The max amount of time (in milliseconds) that PB will wait before sharing a
+  // tracking action with EFF for community learning
+  MAX_CL_WAIT_TIME: 5 * 60 * 1000, // five minutes
```

Comment:
Good call on reducing network load with the reporting timeouts, but I'm curious why 5 minutes?

Reply:
completely arbitrary!